WPMedia/arc-themes-components
# TMEDIA-732: Add utility functions for Byline block

## Ticket

- [TMEDIA-732](https://arcpublishing.atlassian.net/browse/TMEDIA-732)

## Description

Support functions for Byline Block

## Test Steps

1. Checkout branch - `git checkout tmedia-732`
2. Run Storybook `npm run storybook`
3. Check out the formatAuthor and serialJoin storybook documentation

## Author Checklist

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
